From d6711e4675c9cf1eeca0aeac3f84dce100ce9d88 Mon Sep 17 00:00:00 2001
From: Marcel Bargull <marcel.bargull@udo.edu>
Date: Sun, 18 Apr 2021 09:10:18 +0200
Subject: [PATCH 3/4] Makefile: Copy versioned lib files on macOS

Pattern on macOS: <lib>.<version>.dylib rather than <lib>.so.<version>
---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index e616adb..e13d1f8 100644
--- a/Makefile
+++ b/Makefile
@@ -69,8 +69,8 @@ install: $(EXEC) $(LIB)
 	mkdir -p $(DESTDIR)$(BINDIR)
 	mkdir -p $(DESTDIR)$(LIBDIR)
 	cp -a $(EXEC) $(DESTDIR)$(BINDIR)/
-	cp -a $(LIB)* $(DESTDIR)$(LIBDIR)/
-	cp -a $(BAMTOOLS_LIB)* $(DESTDIR)$(LIBDIR)/
+	cp -a $(LIB:$(SHLIB_EXT)=*$(SHLIB_EXT)*) $(DESTDIR)$(LIBDIR)/
+	cp -a $(BAMTOOLS_LIB:$(SHLIB_EXT)=*$(SHLIB_EXT)*) $(DESTDIR)$(LIBDIR)/
 
 clean:
 	rm src/*.o src/subcommands/*.o $(EXEC) $(LIB)
-- 
2.31.1

